tests/command.asterisk.test.js

```diff
@@ -10,6 +10,16 @@ test('when no arguments then asterisk action not called', () => {
   expect(mockAction).not.toHaveBeenCalled();
 });
 
+test('when no arguments with asterisk handler then asterisk action not called', () => {
```

Comment:
Good. 

This test from #844 got broken/lost when I moved the tests to Jest. I didn't understand quite what it was checking. Now after all my research in this issue, I *think* I understand the scenarios, and should add some more tests while it is making some sense!

Reply:
Feel free to add any other tests you can think of.

Reply:
Added lots in Pull Request!